book/from_flat_php_to_symfony2.rst

```diff
@@ -295,8 +305,8 @@ this page introduces even more lingering problems that a framework can solve
 for you. For example, a missing or invalid ``id`` query parameter will cause
 the page to crash. It would be better if this caused a 404 page to be rendered,
 but this can't really be done easily yet. Worse, had you forgotten to clean
-the ``id`` parameter via the ``intval()`` function, your
-entire database would be at risk for an SQL injection attack.
+the ``id`` parameter via the PHP ``intval()`` function, your
```

Comment:
I think the readflow is a bit better if we put "PHP" after the function name: "`intval()` PHP function"